src/app/rundown/components/piece/piece.component.ts

```diff
@@ -11,5 +11,82 @@ export class PieceComponent {
   @Input()
   public piece: Piece
 
+  @Input()
+  public pixelsPerSecond: number
+
+  @Input()
+  public partDuration: number
+
+  @Input()
+  public expectedPartDuration: number
+
+  public DEFAULT_PIECE_DURATION: number = 4000
```

Comment:
Would be nice to have some indication for what unit is used. 
Is it 4000 frames? ms? seconds?

As soon as it has been added to just one of the constants/variables uses in the calculations, then the unit can be inferred for the rest.

Reply:
I take that the unit is the same as for a `Part`, i.e ms?
If so, fell free to mark this one resolved.

Reply:
I have taken your advice and suffixed `_IN_MS` for the Part constant. This one has been removed since it wasn't necessary.